apps/web/components/apps/App.tsx

```diff
@@ -247,7 +247,7 @@ const Component = ({
             t("free_to_use_apps")
           ) : (
             <>
-              {Intl.NumberFormat("en-US", {
+              {Intl.NumberFormat(i18n.language, {
```

Comment:
@emrysal I was going to leave this field undefined so it could detect the browser locale. The problem was if my locale was "en-CA" and the currency was CAD then it wouldn't display the currency code. The `i18n.language` is just the first the language so it will always show the currency code.